packages/wallet/jest/test-setup.ts

```diff
@@ -0,0 +1,25 @@
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+
+// Load environment variables from .env
+import "../config/env";
+import {AddressZero} from "ethers/constants";
+
+Enzyme.configure({adapter: new Adapter()});
+
+// Mock out util functions that require a networkContext
+jest.mock("../src/utils/contract-utils.ts", () => ({
```

Comment:
It might be nicer to set all the contract addresses to `AddressZero` in the `.env` file, along with an explanation that these are replaced by the deploy script. That way seems a little less magic.

Reply:
:+1: